File: 15000/BOJ_15351 인생 점수.java
// https://www.acmicpc.net/problem/15351
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());

        while(n-->0){
            int score = br.readLine().chars()
                .filter(x-> x != ' ')
                .map(x -> x-'A'+1)
                .sum();

            System.out.println(score == 100 ? "PERFECT LIFE" : Integer.toString(score));
        }
    }
}

File: 15000/BOJ_15351 인생 점수.c
// https://www.acmicpc.net/problem/15351
#include <stdio.h>
#include <string.h>

int n;
char str[31];

int main()
{
    scanf("%d\n", &n);
    while(n--) {
        gets(str);

        int score = 0;
        for(int i = 0; i < strlen(str); ++i) {
            score += str[i] != ' ' ? str[i] - 'A' + 1 : 0;
        }

        if(score == 100) printf("PERFECT LIFE\n");
        else printf("%d\n", score);
    }

    return 0;
}